domains/certificates/Query.API/API.IntegrationTests/UnhealthTests.cs

```diff
@@ -2,18 +2,23 @@
 using System.Threading.Tasks;
 using API.IntegrationTests.Factories;
 using FluentAssertions;
+using Testing.Testcontainers;
 using Xunit;
 
 namespace API.IntegrationTests;
 
-public class UnhealthTests : TestBase, IClassFixture<QueryApiWebApplicationFactory>
+public class UnhealthTests :
+    TestBase,
+    IClassFixture<QueryApiWebApplicationFactory>,
+    IClassFixture<PostgresContainer>
 {
     private readonly QueryApiWebApplicationFactory factory;
 
-    public UnhealthTests(QueryApiWebApplicationFactory factory)
+    public UnhealthTests(QueryApiWebApplicationFactory factory, PostgresContainer dbContainer)
```

Comment:
Why this change?

Reply:
This test checks if rabbitmq, returns 503, if it is not available, but due to the fact that rabbitmq uses postgresql, for some outerbox, it requires postgres to be running before it can be started, before we can check rabbitmq's health, so i added a postgresdb container to be spun up.